docs/detections/rules-ui-create.asciidoc

```diff
@@ -108,7 +108,7 @@ You can also leave the *Group by* field undefined. The rule then creates an aler
 +
 IMPORTANT: Alerts created by threshold rules are synthetic alerts that do not resemble the source documents. The alert itself only contains data about the fields that were aggregated over (the *Group by* fields). Other fields are omitted, because they can vary across all source documents that were counted toward the threshold. Additionally, you can reference the actual count of documents that exceeded the threshold from the `kibana.alert.threshold_result.count` field.
 
-. preview:[] (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Select *Suppress alerts* to reduce the number of repeated or duplicate alerts created by the rule. Refer to <<alert-suppression>> for more information.
+. preview::[] (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Select *Suppress alerts* to reduce the number of repeated or duplicate alerts created by the rule. Refer to <<alert-suppression>> for more information.
```

Comment:
The in-line version of the preview tag has to have one colon. This is how it rendered with two:
<img width="811" alt="image" src="https://github.com/elastic/security-docs/assets/87339667/cd772cca-c757-4315-92b2-5c1949e2a2e1">

```suggestion
. preview:[] (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Select *Suppress alerts* to reduce the number of repeated or duplicate alerts created by the rule. Refer to <<alert-suppression>> for more information.
```